src/js/api.ts

```diff
@@ -726,6 +723,7 @@ function makePublicAPI(): PyodideAPI {
   pyodideAPI.ERRNO_CODES = Module.ERRNO_CODES;
   pyodideAPI._module = Module;
   pyodideAPI._api = API;
+  pyodideAPI.RUNTIME_ENV = RUNTIME_ENV;
```

Comment:
Is there a reason we added this to public API here? I don't see any change to `docs/expected_js_docs.txt` so it seems that we didn't document it? The PR calls itself a maintenance PR so it shouldn't be changing public API.